CHANGES/README.rst

```diff
@@ -34,21 +31,22 @@ for the users to understand what it means.
 combined with others, it will be a part of the "news digest"
 telling the readers **what changed** in a specific version of
 the library *since the previous version*. You should also use
-reStructuredText syntax for highlighting code (inline or block),
-linking parts of the docs or external sites.
-If you wish to sign your change, feel free to add ``-- by
+*reStructuredText* syntax for highlighting code (inline or block),
+linking parts of the docs or external sites. However, you do not need to
+reference the issue or PR numbers here as *towncrier* will automatically
```

Comment:
This will need to be copied over into other projects as I'm trying to have this in sync across many repos, including the ones that are not exactly mine.